src/Server.php

```diff
@@ -349,6 +375,12 @@ public function getCacheWithFileExtensions()
      */
     public function getCachePath($path, array $params = [])
     {
+        $customCallable = $this->getCachePathCallable();
+        if ($customCallable instanceof \Closure) {
+            $boundCallable = \Closure::bind($customCallable, $this, self::class);
+
+            return $boundCallable(...func_get_args());
```

Comment:
Why use `func_get_args()` when the 2 arguments `$path` and `$params` are already available?

Reply:
Mostly just to make sure we're passing all the arguments in the same order as the main function, even if more get added.
I can change that, if it doesn't suit the needs.

Reply:
It's just 2 params, let's avoid an unneeded function call :slightly_smiling_face:.

Reply:
:+1: